features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Listing/SampleAPI/SampleAPI.jsx

```diff
@@ -17,43 +17,38 @@
  */
 
 import React, { Component } from 'react';
-
+import { Link } from 'react-router-dom';
 import Typography from '@material-ui/core/Typography';
-import Button from '@material-ui/core/Button';
+import Box from '@material-ui/core/Box';
+import { Link as MUILink } from '@material-ui/core';
+import Grid from '@material-ui/core/Grid';
 import Redirect from 'react-router-dom/Redirect';
 import CircularProgress from '@material-ui/core/CircularProgress';
 import { withStyles } from '@material-ui/core/styles';
 import green from '@material-ui/core/colors/green';
-import Create from '@material-ui/icons/Create';
-import GetApp from '@material-ui/icons/GetApp';
 import { PropTypes } from 'prop-types';
 import { FormattedMessage, injectIntl } from 'react-intl';
-
+import Configurations from 'Config';
 import API from 'AppData/api';
 import Alert from 'AppComponents/Shared/Alert';
-import InlineMessage from 'AppComponents/Shared/InlineMessage';
 import AuthManager from 'AppData/AuthManager';
-import APICreateMenu from '../components/APICreateMenu';
 import getSampleSwagger from './SamplePetStore.js';
 
-const styles = (theme) => ({
+const styles = () => ({
     buttonProgress: {
         color: green[500],
         position: 'relative',
     },
-    headline: {
-        paddingTop: theme.spacing(1.25),
-        paddingLeft: theme.spacing(2.5),
-    },
-    head: {
-        paddingBottom: theme.spacing(2),
-        fontWeight: 200,
-    },
-    content: {
-        paddingBottom: theme.spacing(2),
+    links: {
+        cursor: 'pointer',
     },
-    buttonLeft: {
-        marginRight: theme.spacing(1),
+    root: {
+        '& a': {
+            color: '#34679D',
+        },
+        '& a:visited': {
```

Comment:
@menakaj , @chanaka3d 
I think this will keep the visited color same as the link color

![image](https://user-images.githubusercontent.com/3313885/106588157-7a728480-6570-11eb-9aad-ba5a161fe90e.png)

Reply:
Yes. That was the plan. Otherwise, the UI will have two different colors.

Reply:
+1